Panels/TabbedPanels.js

```diff
@@ -41,11 +46,17 @@ const Tab = Skinnable(TabBase);
 
 const TabGroupBase = kind({
 	name: 'TabGroup',
+
 	styles: {
 		css: componentCss,
 		className: 'tabBar'
 	},
+
 	computed: {
+		className: ({orientation, tabPosition, styler}) => styler.append({
+			horizontal: orientation === 'vertical',
+			positionAfter: tabPosition === 'after'
```

Comment:
These should be refactored to not use specific class names but to use more flexible names:
```
styler.append(orientation, ['position' + cap(tabPosition)])
```
Which will apply the classes `horizontal`, `vertical`, `positionBefore`, and `positionAfter`. These classes may also be available on a higher level already. (Not sure, will have to check.)